frontend/src/components/forms/TooltipIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import classNames from 'classnames';
+import css from './TooltipIcon.module.scss';
+
+export const TooltipIcon: React.FC<{
+  title: string;
+  children?: any;
+}> = ({ title, children }) => {
+  return (
+    <Tooltip
+      classes={{
+        arrow: classNames(css.tooltipArrow),
+        tooltip: classNames(css.tooltip),
+      }}
+      title={title}
+      placement="top"
+      arrow
+    >
```

Comment:
Looking good! I've got a few suggestions for the tooltip component to make it reusable around the project:

I think this component could be called just Tooltip and it could be used in other places where tooltips are needed as well (Customers in task tables, customer stakes in milestones etc.), not just around the info icons. 

`placement="top"` could be also replaced with `placement={placement}` and pass it as parameters just like the title (maybe use top as default). There might be some cases where the placement should be somewhere other than top.

Reply:
It was developed for that specific task. @material-ui/core/Tooltip is already there for other generic usage and is being used already in some other places.